src/js/claims-status/utils/helpers.js

```diff
@@ -243,3 +243,97 @@ export function getCompletedDate(claim) {
 export function getClaimType(claim) {
   return claim.attributes.claimType || 'Disability Compensation';
 }
+
+// TO DO: Replace made up properties and content with real versions once finalized.
+export const STATUS_CONTENT = {
+  nod: 'nod',
+  awaitingHearingDate: 'awaiting_hearing_date',
+  bvaDecision: 'bva_decision',
+};
+
+// TO DO: Replace made up properties and content with real versions once finalized.
+/**
+ * Grabs the matching title and dynamically-generated description for a given current status type
+ * @typedef {Object} Contents
+ * @property {string} title a current status type's title
+ * @property {string} description a short paragraph describing the current status
+ * ----------------------------------------------------------------------------------------------
+ * @param {string} statusType the status type of a claim appeal as returned by the api
+ * @param {Object} details optional, properties vary depending on the status type
+ * @returns {Contents}
+ */
+export function getStatusContents(type, details) {
+  const { nod, awaitingHearingDate, bvaDecision } = STATUS_CONTENT;
+
+  const contents = {};
+  if (type === nod) {
+    const office = details.regionalOffice || 'Regional Office';
+    contents.title = `The ${office} is reviewing your appeal`;
+    contents.description = `The ${office} received your Notice of Disagreement and is revewing 
```

Comment:
🤔  Verification: The newlines and beginning-of-line spaces will be captured in that string, but when it gets put into the `<p>`, it'll get stripped out and appear like we expect it...right?